international-labour-organization.csl

```diff
@@ -137,16 +144,20 @@
         <date-part name="year"/>
       </date>
       <choose>
-        <if type="bill book graphic legal_case legislation motion_picture report song" match="any">
+        <if type="book" match="any">
           <group prefix=" " delimiter=". " suffix=".">
-            <text macro="title"/>
+            <text macro="title" font-style="italic"/>
+            <text variable="collection-title"/>
             <text macro="edition"/>
             <text macro="editor"/>
+            <text variable="volume" form="short" text-case="capitalize-first"/>
             <text macro="publisher"/>
+            <text macro="pages"/>
+            <text variable="number-of-pages"/>
           </group>
         </if>
         <else-if type="paper-conference" match="any">
-          <text macro="title" font-style="italic" prefix=" " suffix=","/>
+          <text variable="title" quotes="false" font-style="normal" font-weight="normal" prefix=" " suffix=","/>
```

Comment:
much easier to read without all the default settings. You can just remove `quotes="false" font-style="normal" font-weight="normal"`